YesterdayApi.UserIdentity.Data/YesterdayApi.UserIdentity.Data.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Please make sure to update all of the packages to 2.2 versions in UserIdentity and Test projects.

Reply:
They are, read the full PR :)